### PR TITLE
Dmp 492 save to temp folder

### PR DESCRIPTION
**Pull request by uchennannamani**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-492

### Change description ###
Implemented DMP-492 ticket. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioFileOperationServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+@ActiveProfiles({"intTest", "h2db"})
+class AudioFileOperationServiceTest {
```
[the remark this answers is not in the document]
Comment (uchennannamani) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioFileOperationServiceTest.java]: code recaptured to have the test for only FileOperationService class

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioFileOperationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    void saveBlobDataToTempWorkspaceTestThree() throws IOException {
+
+        filePath = audioTransformationService.saveBlobDataToTempWorkspace(mediaFile, fileName);
+        assertArrayEquals(mediaFile.toBytes(), Files.readAllBytes(filePath));
+    }
+
+    @AfterEach
+    void deleteFile() throws IOException, InterruptedException {
+        if (filePath != null) {
+            Thread.sleep(1000);     //remove this line before pull request
```
[the remark this answers is not in the document]
Comment (uchennannamani) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioFileOperationServiceTest.java]: Line removed

File: build.gradle [hunk 3]
```diff
@@ -334,6 +334,10 @@ dependencies {
   implementation group: 'org.flywaydb', name: 'flyway-core', version: '9.19.4'
   implementation group: 'io.hypersistence', name: 'hypersistence-utils-hibernate-60', version: '3.5.1'
 
+  // Azure BinaryData Dependencies
+  implementation platform('com.azure:azure-sdk-bom:1.2.13')
+  implementation 'com.azure:azure-storage-blob'
+
```
[the remark this answers is not in the document]
Comment (uchennannamani) [reply build.gradle:340]: Lines 337-339 removed

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/service/FileOperationServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.darts.audio.service;
+
+import com.azure.core.util.BinaryData;
```
Comment (hemantasharma1129) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/service/FileOperationServiceTest.java:3]: test - TBD

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/service/FileOperationServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
```
Comment (hemantasharma1129) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/service/FileOperationServiceTest.java:6]: test2-TBD

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceTest.java [hunk 6]
```diff
@@ -339,4 +349,25 @@ private void createAndLoadMediaEntityGraph() {
         ));
     }
 
+    @Test
+    void shouldSaveAudioBlobDataUsingTempWorkSpace() throws IOException {
+        String fileName = "caseAudioFile.pdf";
+        String tempWorkspace = audioConfigurationProperties.getTempBlobWorkspace();
+        Path filePath = Path.of(tempWorkspace).resolve(fileName);
+
+        when(mockFileOperationService.saveFileToTempWorkspace(
```
Comment (hemantasharma1129) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceTest.java:358]: Test Again

**Review by hemantasharma1129 — APPROVED** [review]

**Pull request merged**